pandas/core/indexes/base.py

```diff
@@ -137,10 +137,9 @@ def _new_Index(cls, d):
 
         return _new_PeriodIndex(cls, **d)
 
-    if issubclass(cls, ABCMultiIndex):
-        if "labels" in d and "codes" not in d:
-            # GH#23752 "labels" kwarg has been replaced with "codes"
-            d["codes"] = d.pop("labels")
```

Comment:
again i think this is worth doing as two lines for codecov purposes

Reply:
@jbrockmendel is that in general, or are there some cases when this is better? As in, shall I revert all the changes of this kind?

Reply:
its a judgement call, i only commented on a few where the coverage info would be most useful

Reply:
OK thanks, I'll try make similar judgement calls with the rest